include/TrustWalletCore/TWCoinType.h

```diff
@@ -104,6 +104,8 @@ enum TWCoinType {
     TWCoinTypeBoba = 10000288,
     TWCoinTypeMetis = 1001088,
     TWCoinTypeAurora = 1323161554,
+    TWCoinTypeEvmos = 10009001,
+    TWCoinTypeEvmosCosmos = 100060,
```

Comment:
Since native Evmos doesn't have slip44 id or int chain id, how about we use "prefix" like this `11800001`? @catenocrypt @rsrbk, the next zone is 2, 3 etc

Reply:
Yes
Great